install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -172,6 +172,9 @@ type WebAppConfig struct {
 	DisableMigration       bool                   `json:"disableMigration"`
 	Usage                  *UsageConfig           `json:"usage,omitempty"`
 	ConfigcatKey           string                 `json:"configcatKey"`
+	ConfigcatBaseUrl       string                 `json:"configcatBaseUrl"`
+	ConfigcatPollInterval  string                 `json:"configcatPollInterval"`
+	ConfigcatDefaultConfig string                 `json:"configcatDefaultConfig"`
```

Comment:
I think we could make this a new `ConfigCat` struct with the individual properties. It would make it slightly easier to reason about the relationship of the properties.